### PR TITLE
KRACOUES-8637:only allow questionnaire copy if a questionnaire has answe...

### PR DESCRIPTION
**Pull request by jwillia**
...rs

### CODE REVIEW

File: coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java [hunk 1]
```diff
@@ -544,6 +545,19 @@ public boolean areActiveQuestionnaires(List<AnswerHeader> answerHeaders) {
         return false;
     }
 
+    public boolean areAnsweredQuestionnaires(List<AnswerHeader> answerHeaders) {
+        for (AnswerHeader answerHeader : answerHeaders) {
+            if (answerHeader.isActive()) {
+                for (Answer answer : answerHeader.getAnswers()) {
+                    if (StringUtils.isNotEmpty(answer.getAnswer())) {
```
Comment (leoherbie) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: I think this is wrong.  Don't all questions have to be answered (ie all answer objects have a non-blank answer).  Also is there existing code you can reuse that determines if a questionaire is complete?

Comment (jwillia) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: seeking more clarification in the jira

Comment (blackcathacker) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: Ohhhh...actually this looks reasonable. All this is doing is determining if the questionnaires can be copied to the new proposal. Not if the questionnaire is complete or not.

Comment (jwillia) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: per jira comment "If you can discern that any questions are answered but the questionnaire is not complete, then by all means, include the option to copy the questionnaire"

Comment (leoherbie) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: ok.  thanks for the clarification with functionals

**Pull request merged**